data/registry_model/registry_proxy_model.py

```diff
@@ -156,6 +215,8 @@ def lookup_manifest_by_digest(
                 )
             except (UpstreamRegistryError, ManifestDoesNotExist) as e:
                 raise ManifestDoesNotExist(str(e))
+            except QuotaExceededException as e:
+                raise e
```

Comment:
consider not catching just to raise it again. and if you do re-raise, use `raise` only (without the `e`)